README.md

```diff
@@ -1,27 +1,64 @@
+3. We use Github (where you are now) to store and manage versions of vocabulary TTL files. Github also provides workflow functionality to approve vocabularies. Read the [Github getting started guide](https://guides.github.com/activities/hello-world/)
+4. We import the TTL files into GraphDB to create a triple store. GraphDB lets us query the triples.
+5. VocPrez presents our vocabs on the web for people and computers to read. VocPrez pulls the triples from GraphDB to create a cache of the vocabularies.
+6. CKAN drop-down form fields pull their values from VocPrez. This ensures that the attributes uses to describe a dataset comes from the controlled vocabulary. 
+
+
+## How to create a vocabulary
+<img src="vocabulary-build-pull-workflow.png" style="width:100%;" alt="Vocabulary build and pull process" />  
+
+**Fig. 2: Vocabulary build and pull process**
+
+1. Select the vocabulary editor of your choice.
+2. Create the vocabulary using the [SKOS Simple Knowledge Organization System](https://www.w3.org/TR/skos-reference/). See also the [SKOS Primer](https://www.w3.org/TR/skos-primer/) for the basics. NOTE: Always first check if there is an international or national vocabulary (see below for links).  
+    a. Use [Vocbench](www.vocbench.gsq.digital:7200/) to create the vocabulary.  
+    b. Use the Excel template to create the vocabulary. *(Vance can you upload the template to this repository and put the link here)*.  
+    c. Edit the vocab TTL file in Visual Studio Code. Use the extension [Language Support for RDF related language syntax](https://marketplace.visualstudio.com/items?itemName=fantasticfears.semantic-data-model-language) for formatting support.  
+3. Export the vocabulary to a TTL file. If using Vocbench, it is easier to export the TTL from the Build repository in GraphDB. Follow the [instructions here](http://graphdb.ontotext.com/documentation/standard/exporting-data.html).
+4. Validate the TTL file using the [online Skosify tool](http://demo.seco.tkk.fi/skosify/skosify). Tick the checkboxes *Keep skos:related relationships within the same hierarchy* and *Include skos:narrower relations in output*
+5. Import the TTL file into a development branch in Github. Name your branch *dev-yourGithubusername*. See [how-to instructions here](https://guides.github.com/activities/hello-world/).
+6. When you're ready to publish your vocabulary into Test, submit a Pull Request to the TEST branch. See [how-to instructions here](https://guides.github.com/activities/hello-world/#pr).
+7. A member of the Data Integrity Team will review your vocabulary and either Approve or Request Changes. See [how-to instructions here](https://help.github.com/en/articles/approving-a-pull-request-with-required-reviews).
+8. If the pull request is approved, the vocab will now be in the TEST branch.
```

Comment:
For approval can formalise apporval steps via pull requests? i.e. something like: creator(loads to)>dev>content authorities(peer review and approve pull request)>review>(Data managers approve data integrity and pull request)>test>(Data masters approve final publish to vocprez)>master